proto/frequenz/api/reporting/v1/reporting.proto

```diff
@@ -296,6 +296,21 @@ message ListMicrogridComponentsDataResponse {
 //     like errors or operational states of the components during the specified time period.
 //
 message ReceiveMicrogridComponentsDataStreamRequest {
+  // Filter for sources for metrics, specified for a single metric at a time.
+  // Specifying a metric here will only stream data for that metric where data
+  // can be found tagged with the given source names.
+  // This is optional, and if omitted, the default metric will be used to stream data.
+  // If a metric is given here but not in the `metrics` field below,
+  // the metric source info will simply be ignored.
+  message MetricSourceOptions {
```

Comment:
We put these messages in the global scope, so that it can be used by other requests as well (e.g., [`IncludeOptions`](https://github.com/frequenz-floss/frequenz-api-reporting/blob/5e033bd49357af126e6ed62782a8f910b6cf6be4/proto/frequenz/api/reporting/v1/reporting.proto#L111-L124)).